safegraph_py_functions/safegraph_py_functions.py

```diff
@@ -225,3 +248,31 @@ def read_pattern_multi(path_to_pattern, compression='gzip', *args, **kwargs):
 def merge_core_pattern(core_df, patterns_df, how='inner', *args, **kwargs):
     merged_df = pd.merge(core_df, patterns_df, on='safegraph_place_id', how=how, *args, **kwargs)
     return merged_df
+
+
+### --------------------------------------- END CORE, GEO, AND PATTERNS SECTION -----------------------------------------------
+
+### --------------------------------------- SOCIAL DISTANCING SECTION -----------------------------------------------
+
+## start_range and end_range = string formated as "year-month-day" 
+    ## ex: start_range = "2020-06-01", end_range = "2020-06-07"
```

Comment:
@bpblakely  In this comment, should these  be called `start_date` and `end_date` instead of `start_range` and `end_range` ?

Reply:
Thanks for the catch